block/provider/create_snapshot_test.go

```diff
@@ -57,106 +56,134 @@ func TestCreateSnapshot(t *testing.T) {
 		verify func(t *testing.T, snapshotResponse *provider.Snapshot, err error)
 	}{
 		{
-			testCaseName: "Not supported yet",
-			providerVolume: &provider.Volume{
-				VolumeID: "16f293bf-test-4bff-816f-e199c0c65db5",
-			},
-			baseVolume: &models.Volume{
-				ID:       "16f293bf-test-4bff-816f-e199c0c65db5",
-				Name:     "test volume name",
-				Status:   models.StatusType("OK"),
-				Capacity: int64(10),
-				Iops:     int64(1000),
+			testCaseName:   "Source VolumeID for snapshot not provided",
+			sourceVolumeID: "",
+			providerSnapshotParameters: &provider.SnapshotParameters{
+				Name: "test snapshot",
 			},
+			providerSnapshot: nil,
 			baseSnapshot: &models.Snapshot{
```

Comment:
Open GHE to track validation of test case data

Reply:
https://github.ibm.com/alchemy-containers/armada-storage/issues/3801